test/classes/initializers/phase2/parentAccess.good

```diff
@@ -1 +1 @@
-{f1 = 1.25, f2 = -1.95, f3 = 1.5625}
+parentAccess.chpl:7: error: Cannot access parent field 'f1' during phase 1
```

Comment:
It might be more appropriate to update the .chpl file so that it doesn't violate the phase 1 rule with the omitted initialization.  This test was intended to demonstrate that it is okay to rely on a parent field in Phase 2, it was an oversight that it inserted a bad access